### PR TITLE
SPCTR-2937 Improvement: Post Grid - Added the option for normal pagination to disable Ajax-based pagination.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2937

### Screenshots

- NA

### Types of changes

- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

**Review by rajmohanreddy774 — APPROVED** [review]

File: blocks-config/post/class-uagb-post.php [hunk 1]
```diff
@@ -1828,6 +1832,9 @@ public function add_post_dynamic_script() {
 
 			if ( isset( self::$settings['grid'] ) && ! empty( self::$settings['grid'] ) ) {
```
Comment (adi3890) [inline blocks-config/post/class-uagb-post.php:1833]: should we check for is_array before foreach

File: blocks-config/post/class-uagb-post.php [hunk 2]
```diff
@@ -1828,6 +1832,9 @@ public function add_post_dynamic_script() {
...
 				foreach ( self::$settings['grid'] as $key => $value ) {
+					if ( 'ajax' !== $value['paginationType'] ) { // Return early when pagination type is not ajax.
+						return;
```
Comment (adi3890) [inline blocks-config/post/class-uagb-post.php:1836]: Since this is in loop should we do continue, 
return will just exit the loop. Can you please elaborate this.

File: src/blocks/post/post-grid/settings.js [hunk 3]
```diff
@@ -604,6 +605,26 @@ const Settings = ( props ) => {
...
+					label={__( 'Pagination Type', 'ultimate-addons-for-gutenberg' )}
+					data={{
+						value: paginationType,
+						label: 'paginationType',
+					}}
+					className="uagb-multi-button-alignment-control"
+					options={[
+						{
+							value: 'ajax',
+							label: 'Ajax',
```
Comment (adi3890) [inline src/blocks/post/post-grid/settings.js:619]: should this labels be translatable

Comment (AbhijitNage123) [reply src/blocks/post/post-grid/settings.js]: Done

File: blocks-config/post/class-uagb-post.php (hunk 1 above)
Comment (AbhijitNage123) [reply blocks-config/post/class-uagb-post.php]: Done

File: blocks-config/post/class-uagb-post.php (hunk 2 above)
Comment (AbhijitNage123) [reply blocks-config/post/class-uagb-post.php]: Yes we can. Basically we don't want run script when PaginationType is Normal.

**Review by aapatre — APPROVED** [review]

File: blocks-config/post/class-uagb-post.php (hunk 2 above)
Comment (AbhijitNage123) [reply blocks-config/post/class-uagb-post.php]: Updated to return early as discussed on call.

**Review by adi3890 — APPROVED** [review]

**Review by aapatre — APPROVED** [review]
